investd/quotes.py

```diff
@@ -35,22 +35,41 @@ def download_quotes_to_csv(
     output_path: Optional[Path] = None,
     start_date: Optional[date] = None,
     end_date: Optional[date] = None,
-    symbols: Optional[Iterable[str]] = [],
+    symbols: Optional[Iterable[str]] = None,
 ) -> None:
     df_tx = load_transactions()
+    symbols = symbols or df_tx["symbol"].unique()
+    adjusted_symbol_to_symbol = {adjust_symbol(symbol): symbol for symbol in symbols}
     df_quotes = fetch_quotes(
-        symbols=symbols or df_tx["symbol"].unique(),
+        symbols=adjusted_symbol_to_symbol.keys(),
         from_date=start_date or df_tx["timestamp"].min().date(),
         until_date=end_date or date.today(),
     )
-    df_quotes.to_csv(output_path or INVESTD_PERSIST / QUOTES_FILENAME)
+
+    # transform yfinance table to simpler format
+    symbol_to_close_column_index = {
+        adjusted_symbol_to_symbol[symbol]: df_quotes.columns.tolist().index(
+            (symbol, "Close")
+        )
+        for symbol in adjusted_symbol_to_symbol.keys()
+        if symbol
+    }
+    pdf = pd.DataFrame.from_dict(
+        {
+            symbol: df_quotes.iloc[:, index]
+            for symbol, index in symbol_to_close_column_index.items()
+        }
+    )
```

Comment:
Wouldn't it be the same if you use `.loc[:, (symbol, "Close")]` instead of `.iloc[:, index]`?

Reply:
Indeed, after two months I have honestly no idea why I did it this way. Refactored.